3.3/.sti/bin/run

```diff
@@ -1,25 +1,112 @@
 #!/bin/bash
 
-function is_gunicorn_installed() {
-    pip show gunicorn
+# This script uses these environment variables:
+#
+# APP_MODULE: Python dotted path to your WSGI application.
+#             If not provided, tries to find the Python path to a 'wsgi.py' file
+#             in the source tree.
+#             That file is present in Django projects by default.
+# APP_CONFIG: Optional configuration file for gunicorn.
+# APP_FILE:   Optional path to Python script to run your application.
+#             Defaults to 'app.py' if it exists.
+#
+# DISABLE_MIGRATE: if not empty, inhibits execution of 'manage.py migrate'.
+
+BIND_ADDR="0.0.0.0:8080"
```

Comment:
This is not an env var, it is a global variable, used by Django's `runserver` and `gunicorn`.